common/src/View/InputEvent.h

```diff
@@ -172,8 +172,8 @@ namespace TrenchBroom {
             Type type;
             Button button;
             WheelAxis wheelAxis;
-            int posX;
-            int posY;
+            double posX;
```

Comment:
we should document the coordinate system and what the units are (points).

Reply:
Done.